### PR TITLE
Update ca.m3u

### PR DESCRIPTION
**Pull request by boomski**
Added 15 Channels

### CODE REVIEW

File: channels/ca.m3u [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+http://ummodigital.com:8000/jorge1/jorge1/5871
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/5n9cgt9.png" group-title="",CBC Windsor
+http://ummodigital.com:8000/jorge1/jorge1/5872
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/bbmuFF4.jpg" group-title="",CBC Winnipeg
+http://ummodigital.com:8000/jorge1/jorge1/5873
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/bbmuFF4.jpg" group-title="",CBC Yellowknife
+http://ummodigital.com:8000/jorge1/jorge1/5874
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/HmqRMIq.png" group-title="Sport",Fubo Cycling
+http://ummodigital.com:8000/jorge1/jorge1/3426
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/k9M5ydW.png" group-title="",5AAB Music Channel
```
Comment (freearhey) [inline channels/ca.m3u]: As I understand it, all links starting with "http://ummodigital.com:8000/jorge1/jorge1" do not work

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/ca.m3u (hunk 1 above)
Comment (adamo331) [reply channels/ca.m3u]: not working. this link from provider and seems was deleted

Comment (boomski) [reply channels/ca.m3u]: Strange, when i open these links in VLC they all play

Comment (freearhey) [reply channels/ca.m3u]: @boomski well what are we going to do? add to playlist or not?

Comment (boomski) [reply channels/ca.m3u]: I tried to open them in kodi and then they also don't play , but for me in vlc they play fine.
But if you or anyone else can't play them i would not add them to the list.

Comment (freearhey) [reply channels/ca.m3u]: Ok, then I close this PR for now.

**Pull request closed without merging**